services/assistant/v1-workspace-manager.js

```diff
@@ -34,6 +36,40 @@ module.exports = function (RED) {
     service = serviceutils.getServiceCreds(OLD_SERVICE_IDENTIFIER);
   }
 
+  const ExecutionList = {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Combine this with the previous 'const' statement.](https://app.codacy.com/manual/BetaWorks-NodeRED-Watson/node-red-node-watson/pullRequest?prid=4846228)